src/Symfony/Bridge/Doctrine/ArgumentResolver/EntityValueResolver.php

```diff
@@ -50,6 +50,7 @@ public function resolve(Request $request, ArgumentMetadata $argument): array
         if (!$options->class || $options->disabled) {
             return [];
         }
+        // doctrine/persistence < 4 compat
```

Comment:
This is not true. `$this->getManager` can still return `null` (see the implementation of the private method). This method call is **not** a call to the doctrine/persistence method.

Reply:
Oh right, didn't spot that!